oggm/core/preprocessing/climate.py

```diff
@@ -136,6 +136,230 @@ def process_custom_climate_data(gdir):
     out = {'climate_source': fpath, 'hydro_yr_0': y0+1, 'hydro_yr_1': y1}
     gdir.write_pickle(out, 'climate_info')
 
+@entity_task(log, writes=['gcm_data'])
+def process_gcm_data(gdir):
+    """Processes and writes the climate data for this glacier.
+
+    Interpolates the GCM data to the high-resolution CL2 climatologies
+    (provided with OGGM) and writes everything to a NetCDF file.
+    """
+    # read the climatology high resolution CL2 climatologies
+    clfile = utils.get_cru_cl_file()
+    ncclim = salem.GeoNetcdf(clfile)
+
+    # GCM temperature and precipitation data
+    if not (('gcm_temp_file' in cfg.PATHS) and \
+                    os.path.exists(cfg.PATHS['gcm_temp_file'])):
+        raise IOError('GCM temp file not found')
+
+    if not (('gcm_precc_file' in cfg.PATHS) and \
+                    os.path.exists(cfg.PATHS['gcm_precc_file'])):
+        raise IOError('GCM precc file not found')
+
+    if not (('gcm_precl_file' in cfg.PATHS) and \
+                    os.path.exists(cfg.PATHS['gcm_precl_file'])):
+        raise IOError('GCM precl file not found')
+
+
+    # read the file
+    fpathTemp = cfg.PATHS['gcm_temp_file']
+    fpathPrecc = cfg.PATHS['gcm_precc_file']
+    fpathPrecl = cfg.PATHS['gcm_precl_file']
+    temp = salem.open_xr_dataset(fpathTemp)
+    temp = temp.TREFHT
+    temp.time
+    temp = temp[9:-3, :, :] # from normal years to hydrological years
```

Comment:
This is ok for now, but good practice would imply that we check if the time series are clean (i.e. here you assume a start in January, which is likely but not guaranteed)